clients/web/src/templates/body/frontPage.pug

```diff
@@ -1,20 +1,20 @@
 .g-frontpage-header
   img.g-frontpage-logo(src=require('girder/assets/Girder_Mark.png'), width="82")
   .g-frontpage-title-container
-    .g-frontpage-title Girder
+    .g-frontpage-title #{brandName}
     .g-frontpage-subtitle Data management platform
 
 .g-frontpage-body
   p.g-frontpage-paragraph
-    b Welcome to Girder!
+    b Welcome to #{brandName}!
     |  If this is your first time here, you might want to check out the
     a(target="_blank", href='http://girder.readthedocs.org/en/latest/user-guide.html')  User Guide
     |  to learn the basics of how to use the application.
     |  To browse the data hosted on this server, start on the
     a.g-collections-link  Collections
     |  page. If you want to search for specific data on this server, use the
     a.g-quicksearch-link  Quick Search
-    |  box at the top of the screen. Developers who want to use the Girder REST API
+    |  box at the top of the screen. Developers who want to use the #{brandName} REST API
```

Comment:
It's true, I think we have to keep some track of 'Girder' even if an admin user can configure lot of other things...